content/en/examples/security/kind-with-namespace-level-baseline-pod-security.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+# Until v1.23 is released, kind node image needs to be built from k/k master branch
+# Ref: https://kind.sigs.k8s.io/docs/user/quick-start/#building-images
```

Comment:
femtonit: the release has happened (very happy to fix this in a follow-up PR, though)